AssetRipperGUI/Localizations/en_US.json

```diff
@@ -50,6 +51,10 @@
   "text_asset_format_text_description": "Export as a plain text file (.txt)",
   "text_asset_format_parse": "Parse",
   "text_asset_format_parse_description": "Export as a plain text file, but try to guess the correct file extension (e.g. JSON files get the .json extension)",
+  "shader_asset_format_dummy": "DummyShader",
+  "shader_asset_format_dummy_description": "Export the shader as a dummy-shader. While preserving data like \"Properties\" or \"FallBack\" it uses a general opaque shader code",
+  "shader_asset_format_asset": "YAML Asset",
+  "shader_asset_format_asset_description": "EXPERIMENTAL! Export the shader as a yaml asset. While this preserves the shader code in a compiled way the Unity Editor will sometimes corrupts the file and it doesn't recognize it as a real shader and \"Shader.Find\" will not work with it.",
```

Comment:
Can you consolidate this description? Maybe something more like this:

Export the shader as a yaml asset. Experimental and only usable for viewing in the editor. The editor can randomly corrupt these files.